apps/jobs/views.py

```diff
@@ -332,15 +335,19 @@ def leaderboard(request, challenge_phase_split_id):
     # while populating leaderboard
     challenge_obj = challenge_phase_split.challenge_phase.challenge
     challenge_hosts_emails = challenge_obj.creator.get_all_challenge_host_email()
+    is_challenge_phase_public = challenge_phase_split.challenge_phase.is_public
+
```

Comment:
Please remove the newline here.

Reply:
Might be a good idea to add a comment here and say what the next line is doing. Something like:
`Exclude the submissions from challenge host team to be displayed on the leaderboard of public phases`